boards/cy8cproto_62_4343_w/Makefile

```diff
@@ -0,0 +1,27 @@
+# Licensed under the Apache License, Version 2.0 or the MIT License.
+# SPDX-License-Identifier: Apache-2.0 OR MIT
+# Copyright OxidOS Automotive 2025 SRL.
+
+include ../Makefile.common
+
+OPENOCD_OPTIONS= -s ${OPENOCD_ROOT}/scripts -f openocd.cfg
+# Define the path to the TBF file to flash an application
+APP=
```

Comment:
Can you make it so this can be overridden on the command line?

Alternatively, we can just have this board use Tockloader's flash-file support. I'm happy to help get this up and running (see #4199 and the "Running Applications" section of the LiteX Arty board for reference).

Reply:
You basically want the `APP=` part to be removed? Or you want something like `APP=<path-to-tbf> make program` to override the default value from the Makefile? You would need to use the `-e` parameter which isn't recommended by the [GNU make manual](https://www.gnu.org/software/make/manual/make.html#Environment). 

As for the second part of the message, I'm not familiar with that option from Tockloader. Do I have to add basic support for my board in Tockloader?